packages/derive/src/lib.rs

```diff
@@ -86,3 +87,75 @@ pub fn entry_point(_attr: TokenStream, mut item: TokenStream) -> TokenStream {
     item.extend(entry);
     item
 }
+
+
+#[proc_macro_attribute]
+pub fn callable_point(_attr: TokenStream, mut item: TokenStream) -> TokenStream {
+    let cloned = item.clone();
+    let function = parse_macro_input!(cloned as syn::ItemFn);
+    let name = function.sig.ident.to_string();
+
+
+    let args_len = function.sig.inputs.len();
+    let arg_types: Vec<&syn::Type> = function.sig.inputs.iter().map(|arg| {
+        match arg {
+            syn::FnArg::Receiver(_) => panic!("callable_point Method type are not allowed."),
```

Comment:
thks! fixed are -> is.
and about prefix `callable_point:`, 
`abort!` prints the macro that caused the error as below. So I didn't put it on.
>    = note: this error originates in the attribute macro `callable_point` (in Nightly builds, run with -Z macro-backtrace for more info)

Reply:
I was mistaken for another way of generating the error. I'll check this again. I want to output the same way as above.

Reply:
It has been improved so that the macro, which is the source of the error, is also output as a note as shown below.


> error: only allowed the "contract_name"
> 
>   = note: this error originates in the attribute macro `dynamic_link`
> 
> 
>   --> src/contract.rs:22:16
>    |
> 22 | #[dynamic_link(contract_nam = "dynamic_callee_contract")]
>    |